From 3f81a6559cd48d8735a7b0e7ca7cedac92db8cc8 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Thu, 23 Mar 2023 09:32:09 +0100
Subject: [PATCH] Find PkgConfig before first call to pkg_check_modules

Follow-up to a0d707014150786ba2e508c144b2bc6b8ad8f316

For some reason this would fail only when disabling Fontconfig.

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 556e017d8..c05e7275e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,6 +76,7 @@ set_package_properties(KF5Baloo PROPERTIES DESCRIPTION "File Searching"
                        PURPOSE "Needed for the File Search runner."
                       )
 
+find_package(PkgConfig REQUIRED)
 pkg_check_modules(QALCULATE libqalculate>2.0 REQUIRED IMPORTED_TARGET)
 
 find_package(KWinDBusInterface CONFIG REQUIRED)
@@ -151,7 +152,6 @@ else()
     set(HAVE_XFIXES 0)
 endif()
 
-find_package(PkgConfig REQUIRED)
 pkg_check_modules(PipeWire IMPORTED_TARGET libpipewire-0.3)
 add_feature_info(PipeWire PipeWire_FOUND "Required for Wayland screencasting")
 
-- 
2.40.0

